### PR TITLE
Feature/paste here post action menu

### PR DESCRIPTION
**Pull request by enidemi**
**Problem:**
Paste Here is not using the post-action menu yet.

**Fix:**
This PR adds post-action menu commands and deletes the original action.

**Commit Details:**
- uses the common parts from paste post-action commands

### CODE REVIEW

File: editor/src/components/canvas/canvas-strategies/post-action-options/post-action-paste.ts [hunk 1]
```diff
@@ -240,7 +298,188 @@ export const PasteWithPropsReplacedPostActionChoice = (
...
+    {
+      elementPaste: elementToPaste,
+      originalContextMetadata: originalMetadata,
+      originalContextElementPathTrees: originalPathTree,
+    },
+    editor.elementPathTree,
+  )
+  if (isLeft(target)) {
+    return target
+  }
```
Comment (Rheeseyb) [inline editor/src/components/canvas/canvas-strategies/post-action-options/post-action-paste.ts:461]: If there's no target we should probably fall back to trying to paste onto the storyboard here

File: editor/src/components/editor/actions/actions.tsx [hunk 2]
```diff
@@ -5608,7 +5471,7 @@ function saveFileInProjectContents(
   }
 }
 
-type ElementToInsert = {
+export type PasteElementToInsert = {
```
Comment (Rheeseyb) [inline editor/src/components/editor/actions/actions.tsx:5474]: It doesn't look like this (or the function below) need to be exported

**Review by Rheeseyb — APPROVED** [review]

File: editor/src/components/editor/actions/actions.tsx (hunk 2 above)
Comment (enidemi) [reply editor/src/components/editor/actions/actions.tsx]: fixed

File: editor/src/components/canvas/canvas-strategies/post-action-options/post-action-paste.ts (hunk 1 above)
Comment (enidemi) [reply editor/src/components/canvas/canvas-strategies/post-action-options/post-action-paste.ts]: good idea, fixed

**Review by bkrmendy — APPROVED** [review]

**Pull request merged**